index.html

```diff
@@ -80,7 +80,109 @@ <h1 class="mn__title">
             /></a>
           </li>
         </ul>
+        <div class="arrow_down">
+        <img
+          src="https://img.icons8.com/ios/45/172B4D/circled-chevron-down.png"
+          alt="circled arrow-down icon"
+        />
+      </div>
       </section>
+      <section class="works__section">
+        <div class="recent__works">
+          <h2>My Recent Works</h2>
+        </div>
+        <div class="bar__under"></div>
+        <div class="proj__container" id="proj__1">
+        <img class="proj__img" src="img/Work 1.png" alt="1st project">
+        <div class="proj__list">
+        <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+        <ul class="proj__lang">
+          <li class="bt"><a href="#">Ruby on rails</a></li>
+          <li class="bt"><a href="#">CSS</a></li>
+          <li class="bt"><a href="#">JavaScript</a></li>
+          <li class="bt"><a href="#">HTML</a></li>
+        </ul>
+        <button class="see__proj__btn">See Project</button>
```

Comment:
- [ ] Good job creating and styling all the buttons inside your project :100:, but as [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) you should add interactions like Active, Hover and Pressed.

![Screenshot from 2023-01-12 19-28-30](https://user-images.githubusercontent.com/105117832/212209397-4926a782-88f0-4414-a8ec-1698e7a84bf2.png)